### PR TITLE
Run kafka and zk restarts in serial when service already running

### PR DESCRIPTION
**Pull request by domenicbove**
# Description

The kafka and zk restart handler logic built out to run in serial when service already started
If not started will run all together

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Tested locally with fresh installs and installs that update the properties files forcing restarts

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: roles/confluent.kafka_broker/tasks/restart_kafka.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+- name: Populate Service Facts
+  service_facts:
+
+- block:
```
Comment (JumaX) [inline roles/confluent.kafka_broker/tasks/restart_kafka.yml:5]: Can you add a comment block that explains how this code works, it's quite counter intuitive to how I think someone would approach this.

Comment (domenicbove) [reply roles/confluent.kafka_broker/tasks/restart_kafka.yml:5]: the docs to me are pretty straight forward (and also easy to find, they are the first result when searching "ansible blocks")
https://docs.ansible.com/ansible/latest/user_guide/playbooks_blocks.html

we dont currently put comments on how modules work, where should we draw the line?

Comment (JumaX) [reply roles/confluent.kafka_broker/tasks/restart_kafka.yml:5]: We should be commenting on complex/not easy to understand pieces of code.  In this instance, it's the "Restart confluent-server Serially" I am referring to.  Not sure my previous comment was clear enough in that regard, apologies.

Comment (domenicbove) [reply roles/confluent.kafka_broker/tasks/restart_kafka.yml:5]: how does it look now?

**Pull request merged**